### ISSUE TITLE
Error: failed pre-install: timed out waiting for the condition

### ISSUE THREAD

**Issue by itsec207** [admirito/gvm-containers#27]
When I tried to install gvm by helm I have some issues and after that installation failed. I don't know why? 

Command:
helm install gvm ./gvm-*.tgz --namespace wazuh--timeout 15m --set gvmd-db.postgresqlPassword="mypassword"

-installed on namespace which I created previosly

Output:

Error: failed pre-install: timed out waiting for the condition

**Comment by admirito** [comment]
There is a pre-install hook that will download the gvm feeds before chart installation that seems to be failing or at least to slow to be finished before 15 minute timeout (maybe the internet connection is slow?).

You can increase the timeout in the command line e.g. `30m` instead of `15m`, but it is better to investigate the problem during the chart installation by looking at the logs of the pre-installation hook pod:

```sh
kubectl get pod

kubectl logs -f <gvm-feeds-sync pod found in the previous command>
```

The warning `skipped value for extraEnv: Not a table` is due to default values in pstgresql chart and is irrelevant.  (Just a warning that you can ignore)

**Comment by admirito** [comment]
Well, it seems that there is a problem with pre-installation hooks for GVM feeds sync and the pre-installation hook pod will be in the `pending` state forever. Can you try again with the new patch in #28 @itsec207?

**Comment by itsec207** [comment]
Hello, 

From logs I saw information   " Warning  FailedScheduling  51s (x10 over 9m15s)  default-scheduler  0/1 nodes are available: 1 persistentvolumeclaim "gvm" not found." I saw that https://github.com/admirito/gvm-containers/releases/download/chart-1.0.1/gvm-1.0.1.tgz not exist

**Comment by admirito** [comment]
Well, I didn't release the binary yet, but you can create the chart manually or use this binary that I just made: [gvm-1.0.1.tar.gz](https://github.com/admirito/gvm-containers/files/6103106/gvm-1.0.1.tar.gz)
```sh
helm install ./gvm-1.0.1.tar.gz ...
```

**Comment by Simon3** [comment]
With the 1.0.1 chart, the gvm-feeds-sync pod is stuck in ContainerCreating state (fresh install): 

```
Events:
  Type     Reason              Age                  From                                       Message
  ----     ------              ----                 ----                                       -------
  Warning  FailedScheduling    12m (x3 over 12m)    default-scheduler                          0/3 nodes are available: 3 pod has unbound immediate PersistentVolumeClaims.
  Normal   Scheduled           12m                  default-scheduler                          Successfully assigned gvm/gvm-feeds-sync-5dls2 to gke-kube-1-n2-pool-29a1f059-5v0g
  Warning  FailedAttachVolume  12m                  attachdetach-controller                    Multi-Attach error for volume "pvc-b0b0559b-a024-4659-9e57-4d6d3a6bbba4" Volume is already used by pod(s) gvm-openvas-75465d577-864jl, gvm-gvmd-68f48968b7-96wfc
  Warning  FailedMount         5m45s (x3 over 10m)  kubelet, gke-kube-1-n2-pool-29a1f059-5v0g  Unable to attach or mount volumes: unmounted volumes=[data-volume], unattached volumes=[run-dir data-volume default-token-6xh96]: timed out waiting for the condition
  Warning  FailedMount         3m29s                kubelet, gke-kube-1-n2-pool-29a1f059-5v0g  Unable to attach or mount volumes: unmounted volumes=[data-volume], unattached volumes=[data-volume default-token-6xh96 run-dir]: timed out waiting for the condition
  Warning  FailedMount         74s                  kubelet, gke-kube-1-n2-pool-29a1f059-5v0g  Unable to attach or mount volumes: unmounted volumes=[data-volume], unattached volumes=[default-token-6xh96 run-dir data-volume]: timed out waiting for the condition
```

**Comment by admirito** [comment]
I have merged the 3926879 that fixes the timeout issue and moved the multi-nodes problem to #34. Thank you @Simon3 for your feedback.
